sportsdataverse/cfb/cfb_pbp.py

```
@@ -5070,6 +4990,389 @@ def __after_cols(self, play_df):
+                            ),
+                        )
+                    )
+                    .and_(pl.col("statYardage") >= pl.col("start.distance"))
+                    # goal-to-go has NO line to gain, so nothing can earn a
+                    # first down there (a goal-to-go TD scores without one) --
+                    # verified vs the ESPN box: e.g. 401032062/BYU is exact
+                    # only when the 4 goal-to-go TDs are excluded.
+                    .and_(pl.col("start.distance") < pl.col("start.yardsToEndzone"))
+                    # a penalty-wiped play earns nothing by yards (the accepted
+                    # penalty path below handles any awarded first down)
+                    .and_(pl.col("penalty_negated_play") == False)
```

Flake8 (7.3.0)

[error] comparison to False should be 'if cond is False:' or 'if not cond:'

(E712)